src/auth/authentication.ts

```diff
@@ -0,0 +1,21 @@
+import { getToken, requestOboToken } from '@navikt/oasis'
+import { NextApiRequest } from 'next'
+
+import { isLocalOrDemo } from '@/constants/envs'
+
+export async function getBearerToken(req: NextApiRequest, backendClientId: string): Promise<string> {
```

Comment:
Vi har en egen funksjon som veksler idporten token til tokenx for meroppfolging-backend som også kalkulerer clientid så vi trenger ikke å legge til dette

Reply:
Ok, endrer! Det ble litt clunky med utropstegn i  const tokenX = await exchangeIdportenTokenForMeroppfolgingBackendTokenx(idportenToken!), men token er sjekket og validert i beskyttetApi - men det vet ikke typescript.

Reply:
Evt gjøre en `if (!idportenToken)` sjekk som lengre nede